src/lib/core/utils/data-element-constraints.ts

```diff
@@ -108,10 +111,14 @@ export function flattenConstraints(
       const value = values[variableName];
       // If a value (variable) has not been user-selected for this constraint, then it is considered to be "in-play"
       if (value == null) return true;
-      // If a constraint does not declare shapes or types, then any value is allowed, thus the constraint is "in-play"
-      if (isEmpty(constraint.allowedShapes) || isEmpty(constraint.allowedTypes))
+      // If a constraint does not declare shapes or types and it allows multivalued variables, then any value is allowed, thus the constraint is "in-play"
+      if (
+        isEmpty(constraint.allowedShapes) &&
+        isEmpty(constraint.allowedTypes) &&
+        constraint.allowMultiValued
+      )
```

Comment:
Do review this please--I thought the existing ORing logic didn't match the comment/context